src/App/Service/Helper/GithubResultToModelTransformer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Console\App\Service\Helper;
```

Comment:
Open to namespace challenging. I don't know if *Helper* is a good name

Reply:
Transformer ?

Reply:
`Transformer` :+1: